docs/conf.py

```diff
@@ -362,7 +366,24 @@ def rstjinja(app, docname, source):
         source[0] = rendered
 
 
+def lol(app, docname, source):
```

Comment:
This function removes the unrendered parts of the jinja templates. I think the name should reflect that.

Reply:
And should include a description